bodhi/server/scripts/approve_testing.py

```diff
@@ -75,10 +92,14 @@ def main(argv=sys.argv):
         testing = db.query(Update).filter_by(status=UpdateStatus.testing,
                                              request=None)
         for update in testing:
-            # If this release does not have any testing requirements, skip it
             if not update.release.mandatory_days_in_testing:
-                print('%s doesn\'t have mandatory days in testing' % update.release.name)
-                continue
+                if not update.autopush:
+                    # If this release does not have any testing requirements and is not autopush,
+                    # skip it
+                    print('%s doesn\'t have mandatory days in testing' % update.release.name)
+                    continue
+                # if the release is configured to autopush, push it to stable.
+                update_to_stable(update, db)
```

Comment:
This will push updates that failed gating.

Reply:
Let's make sure we add a test for this case to ensure that updates that failed gating do not get pushed.

Reply:
Yes good point